op-e2e/actions/interop/dsl/dsl.go

```diff
@@ -374,3 +375,84 @@ func (d *InteropDSL) AdvanceL2ToLastBlockOfOrigin(chain *Chain, l1OriginHeight u
 		d.AddL2Block(chain)
 	}
 }
+
+func (d *InteropDSL) ActSyncSupernode(t helpers.Testing, opts ...actSyncSafeOption) {
```

Comment:
`ActSyncSupernode` assumes that each components advances its head at most a single block. Can we generalize this to multiple blocks?

Reply:
Yes, I originally wanted to avoid because it would add complexity for detecting forks, e.g. if local-safe head updates during the sync it's difficult to tell by asserting on number alone that the fork has happened:

<img width="1045" alt="image" src="https://github.com/user-attachments/assets/c16eb82f-2dbd-4fe4-b60d-f94b3af77d09" />

So we would need to compare the block hash, but then in order to do this we would need to walk back in the tree from the the current unsafe head rather than query by block number (which would defeat the purpose).

I can add this logic so this function is a bit more general though.

Reply:
I opted to remove assertions from ActSyncSupernode and add them instead to a new object {Superchain,Chain}SyncStatusAsserter.

Reply:
Update, this will now be handled in `baseSyncStatusAssertions()` which is called as part of `RequireSeqSyncStatusChanges`